packages/zowe-explorer-api/src/utils/FileManagement.ts

```diff
@@ -34,17 +35,25 @@ export class FileManagement {
     }
 
     public static getZoweDir(): string {
-        return zowe.getZoweDir();
+        const defaultHome = join(homedir(), ".zowe");
+        if (imperative.ImperativeConfig.instance.loadedConfig?.defaultHome !== defaultHome) {
+            imperative.ImperativeConfig.instance.loadedConfig = {
+                name: "zowe",
+                defaultHome,
+                envVariablePrefix: "ZOWE"
+            };
+        }
+        return imperative.ImperativeConfig.instance.cliHome;
```

Comment:
Can we use `imperative.ProfileInfo.getZoweDir()` here since its a static method?

Reply:
Yeah, I forgot to update the PR with the most recent changed 🙏🏽